centralledger/values.yaml

```diff
@@ -49,7 +49,7 @@ containers:
   admin:
     image:
       repository: mojaloop/central-ledger
-      tag: latest
+      tag: v2.0.7.1-pocdb-20180521
```

Comment:
This should be 'latest'

Reply:
Please change to latest as agreed. I will re-publish 'v2.0.7.1-pocdb-20180521' as 'latest'